src/components/NotFoundCard/EagleAnimation.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import styled from 'styled-components'
+import { TweenLite } from 'gsap/TweenMax'
+
+export default class EagleAnimation extends React.Component {
+  componentDidMount() {
+    window.addEventListener('mousemove', this.onMouseMove)
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('mousemove', this.onMouseMove)
+  }
+
+  onMouseMove = event => {
+    const svg = document.querySelector('#svg')
+
+    const mouse = svg.createSVGPoint()
+    const leftEye = this.createEye('#left-eye')
+    const rightEye = this.createEye('#right-eye')
+
+    mouse.x = event.clientX
+    mouse.y = event.clientY
+    requestAnimationFrame(function() {
+      const point = mouse.matrixTransform(svg.getScreenCTM().inverse())
+      leftEye.rotateTo(point)
+      rightEye.rotateTo(point)
+    })
+  }
+
+  createEye = selector => {
```

Comment:
Maybe `createRotationFnForEye` would be more descriptive? `createEye` sounds like it's actually creating the svg element :)